src/openvino/interface.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "../generators.h"
+#include "../search.h"
+#include "interface.h"
+#include "../cpu/interface.h"
+
+namespace Generators {
+namespace OpenVINO {
+
+const char* device_label = "OpenVINO";
```

Comment:
Is this used anywhere now? I think it was only for the DeviceSpan

Reply:
You're right, this is not used anymore. I removed it.